### PR TITLE
Rebase multi-active-gw branch on devel

### PR DESCRIPTION
**Pull request by anfredette**
Rebase multi-active-gw branch on devel

### CODE REVIEW

**Review by Billy99 — APPROVED** [review]

**Comment by anfredette** [comment]
@skitt, does the CODEOWNERS failure have anything to do with [your pr](https://github.com/submariner-io/submariner-operator/pull/1974)?

**Comment by skitt** [comment]
> @skitt, does the CODEOWNERS failure have anything to do with [your pr](https://github.com/submariner-io/submariner-operator/pull/1974)?

Yes, we need that PR before we can merge further changes to the branch.

**Pull request closed without merging**
